File: api-tester-java/src/main/java/com/revenuecat/apitesterjava/EntitlementInfosAPI.java
package com.revenuecat.apitesterjava;

import com.revenuecat.purchases.EntitlementInfo;
import com.revenuecat.purchases.EntitlementInfos;

import java.util.Map;

@SuppressWarnings({"unused"})
final class EntitlementInfosAPI {
    static void check(final EntitlementInfos infos) {
        final Map<String, EntitlementInfo> active = infos.getActive();
        final Map<String, EntitlementInfo> all = infos.getAll();
        final EntitlementInfo i = infos.get("");
    }
}

File: api-tester-kotlin/src/main/java/com/revenuecat/apitesterkotlin/EntitlementInfosAPI.kt
package com.revenuecat.apitesterkotlin

import com.revenuecat.purchases.EntitlementInfo
import com.revenuecat.purchases.EntitlementInfos

@Suppress("unused")
private class EntitlementInfosAPI {
    fun check(infos: EntitlementInfos) {
        val active: Map<String, EntitlementInfo> = infos.active
        val all: Map<String, EntitlementInfo> = infos.all
        val i: EntitlementInfo? = infos[""]
    }
}
